test/javascripts/acceptance/jira-post-menu-test.js

```diff
@@ -8,6 +8,7 @@ acceptance("Jira - post menu", function (needs) {
 
   needs.settings({
     discourse_jira_enabled: true,
+    discourse_jira_allowed_groups: 1,
```

Comment:
A small nitpick, but could you create a constant for the group ID? This would add some clarity.